keps/sig-scheduling/4247-queueinghint/README.md

```diff
@@ -596,7 +596,9 @@ n/a
 - The integration tests are implemented for requeueing scenarios in all plugins.
 - `PreCheck` feature in the scheduling queue is completely removed.
 - No significant degradation in memory comsumption.
-- No performance degradation is confirmed via scheduler_perf.
+- scheduler_perf covers the performance of most QueueingHintFn for in-tree plugins.
+- scheduler_perf runs with QueueingHint both enabled and disabled for all test cases.
+- Event handling duration is monitored using scheduler_perf.
 - The feature gate is enabled by default.
 - No bug report for a while after enabling it by default.
```

Comment:
This should rather be GA criteria, because right now the feature is disabled, so there wouldn't be bug reports.